build/main.js

```diff
@@ -0,0 +1,147 @@
+/*
+ * ATTENTION: The "eval" devtool has been used (maybe by default in mode: "development").
+ * This devtool is neither made for production nor for readable output files.
+ * It uses "eval()" calls to create a separate source file in the browser devtools.
```

Comment:
Kindly, after you completed your entire project, change webpack to production mode and then run the `npm run build ` in order to avoid the use `eval` method according JavaScript best practices.